python/tests/api/writer/test_whylabs_integration.py

```diff
@@ -95,15 +101,16 @@ def test_whylabs_writer_reference(segmented: bool):
     trace_id = str(uuid4())
     result = why.log(df, schema=schema, trace_id=trace_id)
     writer = WhyLabsWriter().option(reference_profile_name="monty")
-    writer.write(result, use_v0=False)
+    success, ref_id = writer.write(result, use_v0=not segmented)
```

Comment:
Can we add a comment on the "not segmented" portion here? I'm not sure I'm following the intention and why the use_v0 is set based on segmentation here.

Reply:
```suggestion
    # TODO: There's some confusion about platform support for segmented reference V0/1 profiles.
    # The test currently fails without use_v0 = !segmented. This may not be the final/intended behavior.
    success, ref_id = writer.write(result, use_v0=not segmented)
```

Reply:
line 105 in your suggestion as the comment is sufficient.